workflow-examples/src/main/java/com/redhat/parodos/examples/vmmigration/dto/models/V1beta1Migration.java

```diff
@@ -0,0 +1,211 @@
+/*
+ * Kubernetes
+ * No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
+ *
+ * The version of the OpenAPI document: v1.21.1
```

Comment:
How was this class generated? should we include the openapi + generating this file as part of our code base or is there an option to rely on external lib for this?

Reply:
Eventually I migrated to use fabric8 to generate the objects, so I had to recreate them from scratch.